02 modal

Silksofthesoul opened the pull request.

Silksofthesoul committed:
Добавлен простой локальный сервер для корректной работы svg спрайтов

Silksofthesoul committed:
node server changes / static resources fix

Silksofthesoul committed:
Убран outline у .button

Silksofthesoul committed:
Подключил ~все доступные css-блоки из заданий? что бы видеть .button

Silksofthesoul committed:
add solution for 05-lection3/02-modal

Silksofthesoul committed:
Merge branch 'master' of github.com:js-tasks-ru/htmlcss-20220120_evgeny-labuzov into 02-modal

Silksofthesoul committed:
fix solution for 05-lection3/02-modal

jsru commented:
Добавляю преподавателя (@GSemikozov) для код-ревью.

Silksofthesoul committed:
fix solution for 05-lection3/02-modal
- убрал не нужное

jsru commented:
Решение было обновлено, посмотрим что скажет @GSemikozov

Silksofthesoul committed:
fix solution for 05-lection3/02-modal
- пришлось переместить кнопку .modal__button-close из .modal_action 
из-за отстутпа снизу в случае если кнопка есть/нет в .modal_action
- из минусов -- при пустом контейнере -- мы его скрываем с отступом, но 
если там будет пробельный симовл -- фокус не срабоатет. Конечно, еще 
можно руками или, скриптом убрать...

jsru commented:
Решение было обновлено, посмотрим что скажет @GSemikozov

Silksofthesoul committed:
fix solution for 05-lection3/02-modal
- rename .modal_action -> .modal__action

jsru commented:
Решение было обновлено, посмотрим что скажет @GSemikozov

GSemikozov commented on 05-lection3/02-modal/modal.css:
```diff
@@ -0,0 +1,134 @@
+  align-items: center;
+}
+
+.modal__fade {
```
чтобы не использовать лишний элемент (это ведь все равно элемент модалки) - можно его удалить, а фон задать через rgba на уровне .modal

Silksofthesoul replied:
Исправлено.

GSemikozov commented on 05-lection3/02-modal/modal.css:
```diff
@@ -0,0 +1,134 @@
+  color: var(--grey-4);
+}
+
+.modal__body p {
```
селекторы тегов заменим на классы)

Silksofthesoul replied:
Исправлено.

Silksofthesoul committed:
fix solution for 05-lection3/02-modal
- add .modal__paragraph
- remove .modal__fade

Silksofthesoul committed:
fix solution for 05-lection3/02-modal
что бы не прыгал контент, при overflow:hidden добавляем полоску размером 
17px (возможно "не прыгать" будет толкько у пользователей chrome под 10й 
виндой)

Silksofthesoul committed:
Merge branch 'master' of github.com:js-tasks-ru/htmlcss-20220120_evgeny-labuzov into 02-modal

jsru commented:
Добавляю преподавателя (@GSemikozov) для код-ревью.
